src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -1957,15 +1957,15 @@ export default class AircraftInstanceModel {
      * @method updateStrip
      */
     updateStrip() {
-        if (this.projected) {
+        if (this.projected || !this.inside_ctr) {
             return;
         }
 
         const heading = heading_to_string(this.mcp.heading);
         const altitude = this.mcp.altitude;
         const speed = this.mcp.speed;
 
-        let destinationDisplay = !this.mcp.isEnabled
+        let destinationDisplay = !this.pilot.hasDepartureClearance
             ? this.destination
             : this.fms.getProcedureAndExitName();
```

Comment:
When I saw this comment originally I was totally ready to fire back with _'This is how it was before'_ until I realized it was that way before because we never had the exit until `caf`.  That isn't a problem now.  Easy update